libkbfs/folder_branch_ops.go

```diff
@@ -2267,7 +2255,13 @@ func (fbo *folderBranchOps) createEntryLocked(
 	// Passthrough journal writes temporarily.
 	if excl == WithExcl {
 		if jServer, err := GetJournalServer(fbo.config); err == nil {
-			wasEnabled, err := jServer.Disable(ctx, fbo.id())
```

Comment:
I did think about that before, but I don't think it works in all cases.  Even if `Disable` thinks that there are no more entries, there might be some caller already partway through `journalBlockServer.archiveBlockReferences()` (e.g., they have a reference to `tlfJournal`, but haven't called `tlfJournal.archiveBlockReferences()` yet), and so the archive references call will fail completely, leading to data loss.  (Same with removes.)

I guess we could retry all of the archive and remove calls if they encounter a shutdown error, but I haven't 100% convinced myself that's foolproof yet.